types/joint.d.ts

```diff
@@ -1243,3 +1340,33 @@ export namespace layout {
         export function layout(graph: dia.Graph | dia.Cell[], options?: LayoutOptions): dia.BBox;
     }
 }
+
+export namespace mvc {
+
+    interface View extends Backbone.View<Backbone.Model>, Backbone.Events {
```

Comment:
At a minimum, I believe this line needs to be:

```TypeScript
interface View<T extends Backbone.Model> extends Backbone.View<T>, Backbone.Events {
```
We still need mvc.View to be generic, because the Paper class needs to extend Backbone.View with the Graph object for its model, not a generic Backbone.Model.

The other concern is that this is not an interface, it is a class.  Why are we defining it as an interface?

```TypeScript
class View<T extends Backbone.Model> extends Backbone.View<T>, Backbone.Events {
```